src/helpers/request.ts

```diff
@@ -99,6 +101,8 @@ export const initMakeRequest: InitMakeRequest = ({
         ...generalFetchOptions,
       };
 
-      return fetch(url, fetchOptions).then(handleFetchResponse(handleResponse));
+      const fetchToUse = providedFetch ?? fetch;
```

Comment:
Do we need to do this when the user can just assign `global.fetch`? That's how `Rx.fromFetch` works IIRC.

Reply:
Users are complaining (https://github.com/unsplash/unsplash-js/issues/123) about polluting the global scope 🤷 . Doesn't cost us much to add it so I figured I'd get it done in this version.

Reply:
Fair enough!